packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -71,22 +73,53 @@ import {
   subscribeResourceSets,
 } from 'xo'
 
+@addSubscriptions(props => ({
+  resourceSet: cb =>
+    subscribeResourceSets(resourceSets =>
+      cb(find(resourceSets, { id: props.resourceSet }))
+    ),
+}))
+@connectStore(() => {
+  return (state, props) => ({
+    isAdmin: isAdmin(state, props),
+    resolvedResourceSet: getResolvedResourceSet(
+      state,
+      props,
+      !props.isAdmin && props.resourceSet !== undefined
+    ),
+  })
+})
 class VifNetwork extends BaseComponent {
+  static propTypes = {
+    resourceSet: PropTypes.string,
+  }
```

Comment:
The subscription decorator will override the resource set ID so this should fail.